projects/gnomad/src/server/server.js

```diff
@@ -2,12 +2,21 @@ import path from 'path'
 
 import compression from 'compression'
 import express from 'express'
+import cookieParser from 'cookie-parser'
+
+const winston = require('winston');
+const logger = winston.createLogger({
+  transports: [
+    new winston.transports.Console(),
+    new winston.transports.File({ filename: '/opt/web/gnomadjs/projects/gnomad/src/server/access.log' })
```

Comment:
```suggestion
    new winston.transports.File({ filename: path.resolve(__dirname, 'access.log') })
```
Full path should not be hard coded... I am not sure but we should be able to use  `path.resolve(__dirname, ...)`